### ISSUE TITLE
react native analytics from segmentio

### ISSUE THREAD

**Issue by sunnymopada** [tonyxiao/react-native-segment-io-analytics#18]
Segment published a library for react native https://github.com/segmentio/analytics-react-native

**Comment by bitpit** [comment]
yeah this should be noted somewhere on the main README page I think, just out of deference to the authors of the original software... I was sitting here scratching my head trying to figure out why we were using this fork of a 3rd party lib before realizing it was because the official API is fairly new!
